src/app/add-member-form/add-member-form.component.html

```diff
@@ -0,0 +1,49 @@
+<form [formGroup]="memberForm" (ngSubmit)="onSubmit()">
+  <label>
```

Comment:
```suggestion
  <label for="firstName">
```
should add "for" for accessibility